### ISSUE TITLE
Download the story archiv? 

### ISSUE THREAD

**Issue by NicolasBae** [instaloader/instaloader#1295]
It is possible to download the story archiv of an account, when the user is logged in?

**Comment by Showmaster3000** [comment]
I think you are searching for the parameter `--highlights` ?

**Comment by NicolasBae** [comment]
No, I don't think so. What I mean is the story archive, which is only visible to the user. It contains the entire story you uploaded, so much more than just the highlights. It is located e.g. in the Android App under your profile -> 3 bars in the top right corner -> archive.
